iac/exception.py

```diff
@@ -14,6 +14,11 @@ class IACErrorCode(Enum):
     INSTANCE_TERMINATION_FAIL = auto()
     NO_SUCH_INSTANCE = auto()
     NO_SUCH_INSTANCE_KIND = auto()
+    INVALID_DOMAIN_NAME = auto()
```

Comment:
Generally, yes, that  is my preference as well.  However, as that was not the convention when I started working on this project, I thought it would be awkward to have some with context and others without.  Let's make the change consistently in this project in a separate PR.   Here is a [card](https://blocky.atlassian.net/browse/BKY-3152?atlOrigin=eyJpIjoiNTRjYzcxYmM3N2VlNGZlZWIzODVhYmNkZWMxOGQyYmIiLCJwIjoiaiJ9)